packages/material-ui/src/FormControl/FormControl.js

```diff
@@ -118,7 +119,8 @@ const FormControl = React.forwardRef(function FormControl(props, ref) {
     return initialFilled;
   });
 
-  const [focused, setFocused] = React.useState(false);
+  const [_focused, setFocused] = React.useState(false);
+  const focused = visuallyFocused ?? _focused;
```

Comment:
Our first usage of ?? in the codebase and the first time I link into what it does: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator. 

Babel transpiles it into 
```js
var focused = (_visuallyFocused = visuallyFocused) !== null && _visuallyFocused !== void 0 ? _visuallyFocused : _focused;
```

What do you think of this approach instead?

```js
const focused = visuallyFocused !== undefined ? visuallyFocused : _focused;
```

Reply:
Done